reference/configuration/assetic.rst

```diff
@@ -50,7 +50,7 @@ Full Default Configuration
                 some_filter:                 []
             workers:
                 # see https://github.com/symfony/AsseticBundle/pull/119
-                # Cache can also be busted via the framework.templating.assets_version
+                # Cache can also be busted via the framework.assets.version
```

Comment:
That's weird, as there is a BC layer: https://github.com/symfony/FrameworkBundle/blob/2.7/DependencyInjection/Configuration.php#L79-113 (triggering a deprecation notice is not breaking)